python/cudf/cudf/core/dataframe.py

```diff
@@ -7425,6 +7425,60 @@ def equals(self, other):
                 return False
         return super().equals(other)
 
+    def explode(self, column, ignore_index=False):
+        """
+        Transform each element of a list-like to a row, replicating index
+        values.
+
+        Parameters
+        ----------
+        column : str or tuple
+            Column to explode. Now only supports one column
```

Comment:
We don't need to indicate it only supports one column, passing a tuple is because names of columns are allowed to be tuples.